connectors/kafka/src/test/java/io/camunda/connector/kafka/integration/KafkaIntegrationTest.java

```diff
@@ -448,7 +454,7 @@ void consumeMessageWithHeaders() {
             kafkaTopic,
             null,
             null,
-            List.of(0L, 0L),
+            null,
```

Comment:
Just to clarify: Why was this change necessary?

Reply:
If we start reading from 0 again, wrong message will be consumed as we are running multiple tests in a certain order. I believe this was added by mistake.